webpack/components/ProxmoxServer/ProxmoxServerCloudinit.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import NetworkInterface from './components/NetworkInterface';
+import { Title, Divider, PageSection, Button } from '@patternfly/react-core';
+import TimesIcon from '@patternfly/react-icons/dist/esm/icons/times-icon';
+import CloudInit from './components/CloudInit';
+
+const ProxmoxServerCloudinit = ({
+  cloudinit,
+  storages,
+  additionalData,
+  from_profile,
+  hasCloudinit,
+  paramScope,
+}) => {
+  const [opts, setOpts] = useState(cloudinit);
+  const [cloudInit, setCloudInit] = useState(false);
+  const [cloudInitData, setCloudInitData] = useState(null);
+  const [nextId, setNextId] = useState(0);
+  useEffect(() => {
+    if (cloudinit && cloudinit.length > 0) {
```

Comment:
```suggestion
    if (cloudinit?.length > 0) {
```

Reply:
removed the cloudinit as it will be separate PR